docker-compose/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: '3.7'
```

Comment:
Any reason to prefer `3.7` to 3 or latest, which appear to be `3.8` ?

Reply:
It unlocks newer Docker features (which this particular project may not be using) at the cost of requiring newer Docker versions. I'll try backing it down. (This change is an artifact of me merging in an adapted version of this compose project that I use.)

Reply:
3.7 would allow this file to specify `init: true` so `tini` is automatically injected as the init process in each container. https://docs.docker.com/compose/compose-file/compose-versioning/#version-37

It might be worth enabling that, since it doesn't seem the images built with Bazel would have that.

Reply:
None of the containers apart from runner do any forking. This means that they don’t need tini.

Reply:
Then will leave it at version `3` then since the file is not using any newer Compose features.

Reply:
> None of the containers apart from runner do any forking. This means that they don’t need tini.

Although `tini` would be helpful in the case of `docker exec`'ing into one of the containers leaving behind a zombie process from a shell. That said, not really an issue with a local deployment like this.